terraform/modules/elasticache/outputs.tf

```diff
@@ -0,0 +1,10 @@
+output "redis_connection_string" {
+  value       = "redis://${aws_elasticache_cluster.main.cache_nodes[0].address}:6379"

```

Comment:
I think I'll parameterise port numbers to avoid any issues were they ever to be changed and forgotten to be updated places like here

Reply:
Port numbers have been made into variables now